api/internal/core/storage/storage_mock.go

```diff
@@ -1,5 +1,19 @@
-// Code generated by mockery v1.0.0. DO NOT EDIT.
-
+/*
```

Comment:
It is a auto-generated file, why not re-generate it.

Reply:
for [license issue](https://github.com/apache/apisix-dashboard/issues/546#issuecomment-718371164), we'd better maintain the mock file manually.